### PR TITLE
Configurator

### PR DESCRIPTION
**Pull request by qrohlf**
boom

see https://github.com/ridewithgps/ridewithgps/pull/2220 for how to try this out

### CODE REVIEW

File: src/widgets/embed-configurator/EmbedConfigurator.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    this.changeTitle = this.changeTitle.bind(this)
+    this.changeDescription = this.changeDescription.bind(this)
+  }
+
+  changeTitle (e) {
+    this.setState({title: e.target.value})
+  }
+
+  changeDescription (e) {
+    console.log('changeDescription')
```
Comment (zackham) [inline src/widgets/embed-configurator/EmbedConfigurator.js]: i imagine you left this in by mistake

File: src/widgets/embed-configurator/EmbedConfigurator.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  embedIframe () {
+    const {eventId} = this.props
+    const {title, description} = this.state
+    const params = {eventId, title, description}
+    const iframeStyle = 'width: 100%; height: 530px; border: none;'
+    const query = Object.keys(params)
+      .filter(key => params[key])
+      .map(key => `${key}=${escape(params[key])}`).join('&')
```
Comment (zackham) [inline src/widgets/embed-configurator/EmbedConfigurator.js]: i think you want `encodeURIComponent` instead of `escape`

**Comment by zackham** [comment]
aaand i think my last merge introduced conflicts 🎉

**Comment by qrohlf** [comment]
retest this please

**Comment by qrohlf** [comment]
ok jenkins

**Comment by qrohlf** [comment]
ok jenkins

**Comment by qrohlf** [comment]
retest this please

**Comment by qrohlf** [comment]
@zackham above test result is bogus because Jenkins is being dumb, but I ran the tests locally and they all pass.

**Pull request merged**